connection_test.go

```diff
@@ -970,3 +970,49 @@ func TestPeerStatusChangeServer(t *testing.T) {
 	})
 	assert.Len(t, changes, 0, "unexpected peer status changes")
 }
+
+func TestContextCanceledOnClientTCPClose(t *testing.T) {
+
+	opts := testutils.NewOpts().DisableLogVerification().NoRelay()
```

Comment:
do you need log verification? Please add a comment about this line for why these are disabled.